### PR TITLE
Query suggestions should search for keywords

### LINKED ISSUE #878: Query Sugestions do campo busca consultar também as palavras-chave
No query sugestions hoje não consulta na base de palavras-chave, no entanto esse campo se constitui um dos mais importantes campos de indexação da base de dados. O query sugestions precisa buscar também nesse campo.

### PR DESCRIPTION
**Pull request by mateus4k**
## Summary

Resolves #878

## Checklist

- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/web/components/Algolia/searchBox.js [hunk 1]
```diff
@@ -45,23 +38,9 @@ const SearchBox = ({ placeholder, onChange, onSubmit, currentRefinement, refine,
 				inputProps={{ placeholder, onChange: handleChange, value: inputValue }}
 				renderSuggestionsContainer={({ containerProps, children }) => (
 					<StyledSuggestionsContainer>
-						<StyledStats
-							translations={{
```
Comment (mateus4k) [inline packages/web/components/Algolia/searchBox.js:49]: I'm sorry for that, @luizeboli 😢

File: packages/api/app/Utils/Algolia/indexes/service.js [hunk 2]
```diff
@@ -13,6 +13,16 @@ const SERVICE_TYPES = {
...
+/**
+ * Returns an array of keywords
+ *
+ * @param {object} service The service object
+ * @returns {string[]} The service keywords
+ */
+const normalizeKeywords = (service) => {
+	return service.keywords.map((keyword) => keyword.term);
+};
+
```
Comment (luizeboli) [inline packages/api/app/Utils/Algolia/indexes/service.js:25]: `keywords` belongs to many solutions, what about move it to `core` or somewhere else so we can reuse easily?

File: packages/api/app/Utils/Algolia/indexes/technology.js [hunk 3]
```diff
@@ -67,6 +67,16 @@ const normalizeCosts = (technology) => {
...
+/**
+ * Returns an array of keywords
+ *
+ * @param {object} technology The technology object
+ * @returns {string[]} The technology keywords
+ */
+const normalizeKeywords = (technology) => {
+	return technology.keywords.map((keyword) => keyword.term);
+};
+
```
Comment (luizeboli) [inline packages/api/app/Utils/Algolia/indexes/technology.js:79]: So we don't duplicate code heh

**Review by luizeboli — CHANGES_REQUESTED** [review]
Everything's working fine but Service's search.

You forgot to add Keywords filter to the sidebar:

image

Also added one comment \o/

**Review by luizeboli — APPROVED** [review]
🎉

**Pull request merged**